packages/engine/Specs/Scene/PickingSpec.js

```diff
@@ -134,6 +134,10 @@ describe(
     function createTileset(url) {
       const options = {
         maximumScreenSpaceError: 0,
+        // The camera is zoomed pretty far out for these tests, so
```

Comment:
Hmm, I wouldn't expect this to be needed. Why is this needed for 3D Tiles but not terrain?

Reply:
the problem seemed to be a race condition when waiting for the tileset to load in this particular test setup. I opened https://github.com/CesiumGS/cesium/issues/11732 for this since it only seems to affect the unit tests, not the rendering itself.